app/konnect/getting-started/add-api.md

```diff
@@ -20,7 +20,9 @@ In this guide, you will build on your Konnect environment by adding your own API
 
 ## Prerequisites
 
-If you didn't use the in-app onboarding wizard in {{site.konnect_short_name}}, open the Konnect [Gateway Manager](https://cloud.konghq.com/gateway-manager), and create a new data plane node.
+* [A {{site.konnect_short_name}} account](https://konghq.com/products/kong-konnect/register)
```

Comment:
@connorckong I added links to the Konnect registration link to the prereqs here and on the import page. I also added a tile to the Konnect overview page with a link as well. I did this instead of making a separate "Create a Konnect account" page because creating an account is pretty easy and doesn't require documentation.

Reply:
Could you use this link instead? https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=konnect-getting-started